file_fetcher.ts

```diff
@@ -123,7 +123,7 @@ export class FileFetcher {
     const content = await response.text();
     const headers: Record<string, string> = {};
     for (const [key, value] of response.headers) {
-      headers[key.toLowerCase(), value];
+      headers[key.toLowerCase()] = value;
```

Comment:
Should we be lowercasing these here? https://github.com/denoland/deno/issues/12829

Reply:
So the `http_util` uses `.to_string()` in on the `HeaderName` from the HTTP request: https://docs.rs/reqwest/0.11.7/reqwest/header/index.html.  I believe this lowercases the header, irrespective of how it was sent across the wire, so when we serialize it to disk, we need to replicate the behaviour of the CLI code.